discord/user.py

```diff
@@ -115,7 +115,7 @@ def __str__(self) -> str:
             if not self.is_migrated
             else (
                 f"{self.name} ({self.global_name})"
-                if self.global_name is not None
+                if self.global_name is not None and self.name != self.global_name
```

Comment:
Could you explain the reason behind this change?

Reply:
If we look to a similar system of nicknames vs. usernames, if you try to set a nickname equal to your username it will instead reset to `None`. But at the moment, it is possible to set a `global_name` equal to your username with no other effects.
While this is fine in itself (though i'm not sure this is intentional), it's somewhat awkward in print; e.g. if i set both my (migrated) `username` and `global_name` to "neloblivion", it would print as `neloblivion (neloblivion)`. That being said, If you think this change is unnecessary then feel free to discard it.

Reply:
Imo it's ok, what u think @Dorukyum

Reply:
I see no issue with returning "dorukyum (dorukyum)" if these two fields have the same value. At least there's a consistent style that way.

Reply:
oke